src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -70,11 +82,16 @@ public JsonObject getTsnePlotData(String experimentAccession) {
 
         result.addProperty("suggesterEndpoint", "json/suggestions");
 
-        result.add(
-                "anatomogram",
-                GSON.toJsonTree(
-                        ontologyAccessionsSearchService.searchAvailableAnnotationsForOrganAnatomogram(
-                                experimentAccession)));
+        if (EXPERIMENTS_WITH_NO_ANATOMOGRAM.contains(experimentAccession)) {
+            result.add(
+                    "anatomogram", GSON.toJsonTree(""));
+        } else {
+            result.add(
+                    "anatomogram",
+                    GSON.toJsonTree(
+                            ontologyAccessionsSearchService.searchAvailableAnnotationsForOrganAnatomogram(
+                                    experimentAccession)));
+        }
```

Comment:
Remember that you can rewrite this as:
```java
result.add(
        "anatomogram",
        GSON.toJsonTree(EXPERIMENTS_WITH_NO_ANATOMOGRAM.contains(experimentAccession) ?
                "" :
                ontologyAccessionsSearchService.searchAvailableAnnotationsForOrganAnatomogram(experimentAccession));
```

Now, I’m not saying that you should change it, but consider it before merging and write it however it feels more expressive and easier to understand to you.

Reply:
Sounds good Alfonso. I've changed it according to the suggested style. It looks much cleaner and also avoids code duplication.